test/dotnet-build.Tests/IncrementalTestBase.cs

```diff
@@ -12,6 +12,9 @@ namespace Microsoft.DotNet.Tools.Builder.Tests
 {
     public class IncrementalTestBase : TestBase
     {
+        protected readonly string _libraryFrameworkFullName = ".NETStandard,Version=v1.5";
+        protected readonly string _appFrameworkFullName = ".NETCoreApp,Version=v1.0";
```

Comment:
Oh nice, I did not know this. Thanks for the explanation!

Since it's not critical I won't kick off another build for this change, but if CI comes back red will make the change.